paths/electron/starting-with-electron/03-git-repository.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Creating a Git Repository
+permalink: /electron/starting-with-electron/init-git-repository
+next-page: /electron/starting-with-electron/create-app-files
+facilitator: false
+sidebar:
+  nav: "electron-getting-starting"
+main-content: |
+
+  We have a great start of an Electron app. But, it's not being tracked with version control. Now, it's time to track this repository with Git, include a .gitignore file that fits this project, and push this up to a remote on GitHub.
```

Comment:
@brianamarie `electron-forge` initialized a repo for me:

```bash
$ electron-forge init electron-app
✔ Checking your system
✔ Initializing Project Directory
✔ Initializing Git Repository
✔ Copying Starter Files
✔ Initializing NPM Module
✔ Installing NPM Dependencies
```

```bash
$ ls -al
total 32
drwxr-xr-x    9 hectorsector  staff    306 Aug  4 17:03 .
drwxr-xr-x    4 hectorsector  staff    136 Aug  4 17:03 ..
-rw-r--r--    1 hectorsector  staff    756 Aug  4 17:03 .compilerc
-rw-r--r--    1 hectorsector  staff    216 Aug  4 17:03 .eslintrc
drwxr-xr-x    9 hectorsector  staff    306 Aug  4 17:03 .git
-rw-r--r--    1 hectorsector  staff     17 Aug  4 17:03 .gitignore
drwxr-xr-x  563 hectorsector  staff  19142 Aug  4 17:03 node_modules
-rw-r--r--    1 hectorsector  staff   1459 Aug  4 17:03 package.json
drwxr-xr-x    4 hectorsector  staff    136 Aug  4 17:03 src
```

If we should remove this, I'm happy to commit it directly.